libs/utils/src/lib/github-PR-service/github.service.ts

```diff
@@ -0,0 +1,119 @@
+import { Injectable } from '@angular/core';
+import { GitHub } from 'github-api';
+import { HttpClient } from '@angular/common/http';
+import { AngularFireAuth } from '@angular/fire/auth';
+import * as firebase from 'firebase/app';
+import { AngularFireDatabase } from '@angular/fire/database';
+
+@Injectable()
+export class GithubService {
+  repo = 'codelab-fun/codelab';
+  githubAuth;
+
+  constructor(
+    private http: HttpClient,
+    private afAuth: AngularFireAuth,
+    private database: AngularFireDatabase
+  ) {
+    afAuth.authState.subscribe(authData => {
+      if (authData === null) {
+        this.login();
+      } else {
+        this.githubAuth = authData;
+      }
+    });
+  }
+
+  async login() {
+    const provider = new firebase.auth.GithubAuthProvider().addScope('repo');
+    this.githubAuth = await this.afAuth.auth.signInWithPopup(provider);
+  }
+
+  async createIssue(message) {
+
+    if (!this.githubAuth.credential) {
+      await this.login();
+    }
+
+    const issueData = {
+      title: message.comment.substring(0, 150),
```

Comment:
hm, why are we doing substring here? 
Is there limit on the field? If not, let's drop it